build_tools/mako/configuration.py

```diff
@@ -118,30 +131,46 @@ def get_s20_default_target_list():
                      "--iree-vulkan-target-triple=valhall-g77-unknown-android10"
                  ])
   ]
+  for target in targets:
+    if target.mako_tag in batch_config:
+      target.add_batch_flag(batch_config[target.mako_tag])
+  return targets
 
 
+# The batch numbers are roughly computed to let it benchmark more than 3
+# seconds.
+# Do not set batch size on Pixel 4 for GPU targets, because it will get killed
```

Comment:
We'll see fluctuation for Pixel 4 then. Is there a way to disable mako upload failure for Pixel 4 GPU?

Reply:
Do you mean to disable the regression analyzer but still uploading the numbers? Or you mean to disable the whole benchmarking for Pixel4 GPU?

I can make the former one work by tweaking the uploader program, this should not be hard. The latter one is super easy, removing the target is enough.